Umbraco-Cloud/Set-Up/Manage-Domains/Umbraco-Latch.md

```diff
@@ -0,0 +1,21 @@
+# Umbraco Latch
+
+All new projects on Umbraco Cloud are automatically protected by Umbraco Latch. This means, that the default Umbraco Cloud URL for your project as well as any new hostnames you add will be assigned a TLS certificate automatically.
+
+![Adding a hostname](images/adding-hostname-to-cloud.gif)
+
+In order for Umbraco Latch to be applied to your hostname, you need to make sure you have either of the following:
+
+* DNS has been setup with a CNAME pointing at the Cloud URL (mysite.s1.umbraco.io)
+* DNS has been setup with an A-Record pointing at the Cloud IP (23.100.15.180)
```

Comment:
Should we omit this, if it is not officially supported?

Reply:
Might be some useful info for you to know if a customer is wanting to point root records at Cloud, a DNS provider called DNSMadeEasy has created a type of record they call ANAME. This allows you to have a alias (CNAME) for root records. https://dnsmadeeasy.com/services/anamerecords/ Super handy!